test-case/check-pause-resume.sh

```diff
@@ -79,15 +90,15 @@ spawn $cmd -D $dev -r $rate -c $channel -f $fmt -vv -i $file_name -q
 set i 1
 expect {
     "*#*+*\%" {
-        set sleep_t [expr int([expr rand() * 1000]) + 1000 ]
-        puts "\r(\$i/\$repeat_count) Wait for \$sleep_t ms for pause"
+        set sleep_t [expr int([expr rand() * $rnd_range]) + $rnd_min ]
+        puts "\r(\$i/$repeat_count) Wait for \$sleep_t ms for pause"
```

Comment:
change "Wait for \$sleep_t ms for pause"  to "Wait for \$sleep_t ms before pause"